app/builders/digest_email_builder.rb

```diff
@@ -21,60 +24,53 @@ def call
 
   def body
     <<~BODY
-      #{I18n.t!("emails.digests.#{digest_run.range}.opening_line")}
+      Update from GOV.UK for:
 
-      #{presented_results}
-      #{I18n.t!("emails.digests.#{digest_run.range}.permission_reminder")}
+      #{title_and_optional_description}
 
-      #{ManageSubscriptionsLinkPresenter.call(address)}
-    BODY
-  end
+      ---
 
-  def presented_results
-    presented_segment(digest_item)
-  end
+      # #{I18n.t("emails.digests.#{digest_run.range}.heading")}
 
-  def presented_segment(segment)
-    <<~RESULT
-      #{presented_header(segment)}
+      #{presented_results}
 
-      #{presented_content(segment.content)}
       ---
 
-      #{UnsubscribeLinkPresenter.call(segment.subscription_id, segment.subscriber_list_title)}
-    RESULT
-  end
+      # Why am I getting this email?
 
-  def presented_header(segment)
-    copy = "# #{presented_title(segment)} &nbsp;"
+      #{I18n.t("emails.digests.#{digest_run.range}.footer_explanation")}
 
-    if segment.subscriber_list_description.present?
-      copy += "\n\n#{segment.subscriber_list_description}"
-    end
+      #{digest_item.subscriber_list_title}
 
-    copy
+      [Unsubscribe](#{unsubscribe_url})
+
+      [#{I18n.t!('emails.digests.footer_manage')}](#{PublicUrls.authenticate_url(address: address)})
+    BODY
   end
 
-  def presented_title(segment)
-    if segment.subscriber_list_url
-      "[#{segment.subscriber_list_title}](#{Plek.new.website_root}#{segment.subscriber_list_url})"
-    else
-      segment.subscriber_list_title
+  def presented_results
+    changes = digest_item.content.map do |item|
+      presenter = "#{item.class.name}Presenter".constantize
+      presenter.call(item, frequency: digest_run.range)
     end
+
+    changes.join("\n---\n\n").strip
   end
 
-  def presented_content(content)
-    changes = content.map do |item|
-      case item
-      when ContentChange
-        ContentChangePresenter.call(item, frequency: digest_run.range)
-      when Message
-        MessagePresenter.call(item, frequency: digest_run.range)
-      else
-        raise "Unexpected content type: #{item.class}"
-      end
+  def title_and_optional_description
+    result = "# #{digest_item.subscriber_list_title}"
+
+    if digest_item.subscriber_list_description.present?
+      result += "\n\n" + digest_item.subscriber_list_description
```

Comment:
is it worth keeping the string construction consistent here? We've got `"# #{digest_item.subscriber_list_title}"` and  `"\n\n" + digest_item.subscriber_list_description` a couple of lines apart

Reply:
Could you explain a bit more what you mean here? I'm not following, after reading a couple times.

Reply:
I sort of meant that we should either have:

Reply:
Oh, I see what you mean. Sure, I've amended the last commit to be more consistent.